mlos_bench/mlos_bench/optimizers/convert_configspace.py

```diff
@@ -97,5 +130,91 @@ def tunable_values_to_configuration(tunables: TunableGroups) -> Configuration:
     Configuration
         A ConfigSpace Configuration.
     """
+    values: Dict[str, TunableValue] = {}
+    for (tunable, _group) in tunables:
+        if tunable.special:
+            (special_name, type_name) = special_param_names(tunable.name)
+            if tunable.value in tunable.special:
+                values[type_name] = "special"
```

Comment:
Maybe make an `Enum` class for "special" and "range" static strings for better checking enforcement at dev time.

Reply:
Done. #635

Reply:
done